src/wiki/useful-windows-tips.md

```diff
@@ -269,12 +269,12 @@ Every so often you'd find yourself in a pinch when the C drive runs low on space
 
    ![Hiberfile 40%](./img/useful-windows-tips/pwrcfg40.png)
 
-   If you do not wish to use hibernation, but still want to use Fast Startup (e.g. you have an HDD as your boot drive) then use `powercfg /h /type reduced` instead, which will set the hiberfile size to 20% of its original size (see the difference). Note that this disables hibernation.
+   If you do not wish to use hibernation, but still want to use Fast Startup (e.g. you have an HDD as your boot drive) then use <code>powercfg /h /type reduced</code> instead, which will set the hiberfile size to 20% of its original size (see the difference). Note that this disables hibernation.
```

Comment:
Why? You changed this one but not the others, and I don't understand what difference it makes.